crates/wash-lib/src/cli/output.rs

```diff
@@ -39,7 +39,7 @@ pub struct StopCommandOutput {
 /// JSON output representation of the `wash link query` command
 #[derive(Debug, Deserialize)]
 pub struct LinkQueryCommandOutput {
-    pub links: Vec<HashMap<String, ActorLinks>>,
+    pub links: Vec<HashMap<LinkName, Vec<InterfaceLinkDefinition>>>,
```

Comment:
Did this work for you? I _think_ that now this should just be a Vec<InterfaceLinkDefinition>, but we can approach that when testing wash